api/blueprints/drbCitation.py

```diff
@@ -59,6 +60,777 @@ def citationFetch(uuid):
 
 def mlaGenerator(citationWork):
     if citationWork:
-        return f'A FORMATTED CITATION {citationWork.title}'
+        if citationWork.title == "The Bible":
+            return mlaBibleGenerator(citationWork)
+
+        elif citationWork.editions[0].measurements:
+            if citationWork.editions[0].measurements[0] == {"type": "government_document", "value": "1"}:
+                return mlaGovGenerator(citationWork)
+
+        elif len(citationWork.authors) == 0:
+            return mlaNoAuthorGenerator(citationWork)
+
+        elif len(citationWork.authors) == 1:
+            return mlaOneAuthorGenerator(citationWork)
+
+        elif len(citationWork.authors) == 2:
+            return mlaTwoAuthorsGenerator(citationWork)
+
+        elif len(citationWork.authors) == 3:
+            return mlaThreeAuthorsGenerator(citationWork)
+            
+        else:
+            return "MLA Citation not avaliable"
+    else:
+        return "MLA Citation not avaliable"
+
+def mlaBibleGenerator(citationWork):
+    #Bible editions before 1900
+        #Specific version info of Bible currently not displayed in database
+    if citationWork.editions[0].publication_date < date(1900, 1, 1):
+        citationResponse = f"{citationWork.title}. Unknown version, \
+                            {citationWork.editions[0].publication_place}, \
+                            {citationWork.editions[0].publication_date}."
+            
+        return ' '.join(citationResponse.split())
+
+    #Bible editions during and after 1900
+        #Specific version info of Bible currently not displayed in database
+    else:      
+        citationResponse = f"{citationWork.title}. Unknown version, \
+                            {citationWork.editions[0].publishers[0]['name']}, \
+                            {citationWork.editions[0].publication_date}."
+            
+        return ' '.join(citationResponse.split())
+
+def mlaGovGenerator(citationWork):
+    #Government Publications before 1900
```

Comment:
Could you add a comment at the top of this describing why 1900 is the dividing line for these records? Very useful to know, would be good to know what's the difference! (looks like Publisher name vs place?)